app/src/main/java/org/simple/clinic/security/pin/PinEntryModel.kt

```diff
@@ -2,16 +2,18 @@ package org.simple.clinic.security.pin
 
 import android.os.Parcelable
 import kotlinx.android.parcel.Parcelize
+import org.simple.clinic.security.pin.PinEntryCardView.State.PinEntry
 
 @Parcelize
 data class PinEntryModel(
+    val state: PinEntryCardView.State,
```

Comment:
I called it `Mode` since that is essentially what it is.

I also ended up removing the property from `PinEntryModel` since it is only being set through `UiActions`.